components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/dao/HybridConfigDAOImpl.java

```diff
@@ -37,17 +38,18 @@ public class HybridConfigDAOImpl implements ConfigDAO {
     public String getGlobalPolicyAlgorithmName() {
 
         // TODO: revisit in caching, make sure it's not skipped
-        String algorithm = jdbcConfigDAO.getPolicyCombiningAlgorithm();
+        int tenantId = CarbonContext.getThreadLocalCarbonContext().getTenantId();
+        String algorithm = jdbcConfigDAO.getPolicyCombiningAlgorithm(tenantId);
```

Comment:
if we have a `jdbcConfigDAO.getPolicyCombiningAlgorithm()` method (without tenantid) param lets use that one here..

In add its use method without tenantid and here use a method with an tenantid.. Lets make it consistent.

Reply:
the respective method `getGlobalPolicyAlgorithmName()` returns with a default value, right. Then it should be fine.